documentation/asciidoc/computers/getting-started/configuring.adoc

```diff
@@ -1,49 +1,63 @@
-== Configuration on First Boot
+== Initial setup and configuration
 
-If you have not already configured your operating system using the xref:getting-started.adoc#advanced-options[Advanced Menu] of Raspberry Pi Imager when Raspberry Pi OS starts up for the first time, you will be guided through initial setup.
+When you start your Raspberry Pi for the first time after installing an operating system, a configuration wizard will guide you through the system's initial setup.
```

Comment:
I'm wondering if this should say `Raspberry Pi OS` rather than `operating system`?